include/triton/Dialect/TritonNvidiaGPU/IR/TritonNvidiaGPUOps.td

```diff
@@ -307,18 +314,19 @@ def TTNG_AsyncTMACopyLocalToGlobalOp : TTNG_Op<"async_tma_copy_local_to_global",
     by `desc_ptr`.
   }];
 
-  let arguments = (
-    ins TT_PtrType:$desc_ptr,
+  let arguments = (ins
+    Arg<TT_PtrType, "", [MemWrite<GlobalMemory>]>:$desc_ptr,
```

Comment:
We read from the descriptor pointer, and write to an unknown pointer contained within the descriptor. Not sure how you would represent that.
```suggestion
    Arg<TT_PtrType, "", [MemRead<GlobalMemory>]>:$desc_ptr,
```

Reply:
If it is contained in the descriptor, I think it makes sense to consider it a write on the descriptor itself, and then we consider the descriptor as aliasing that unknown location. 

The alternative is to consider the op as writing arbitrary global memory, but we know that access to the memory behind two different descriptors do not overlap. I think considering descriptors (and kernel arguments in general) as restrict is a decent assumption. Without it, pipelining wouldn't work anyways

Reply:
Changed this to indicate a read and write effect to global memory on the descriptor operand.

Reply:
@Mogball you changed the wrong operator, you changed the functional API rather than the ttng operators.

Reply:
Oops. Let me send another PR to fix them as well.